packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts

```diff
@@ -112,18 +113,32 @@ export default class TableResize implements EditorPlugin {
             this.cacheRects();
         }
 
+        this.isRTL = getComputedStyle(this.editor.getDocument().body, 'direction') == 'rtl';
```

Comment:
Since you calculate this value everytime when mouse move, why need to cache it?

Actually you can cache it, but only calculate it when a new resize session is starting, e.g., move this line up to L114.

Reply:
Good question! I was about to ask if we can add this logic into the **initialize** method instead, that way, we only calculate this value once, but I did not make that change since we cannot test this feature that way. What do you think?
```
 
initialize(editor: IEditor) {
        this.editor = editor;
       this.isRTL = getComputedStyle(this.editor.getDocument().body, 'direction') == 'rtl';
        ....
    }
```

Reply:
Calculate during initialize may not be good since RTL can be changed during editing. You can calculate it each time the inserter is added, then during the mouse move no need to calculate again.